scripts/count_opcodes.py

```diff
@@ -84,8 +92,38 @@ def find_loops(co):
     return loops
 
 
+class CacheCounter:
+    def __init__(self, counter):
+        self.counter = counter
+        self.offset = 0
+        self.index = 0
+        self.prev_is_extended_arg = False
+
+    def update_offset(self, op):
```

Comment:
I think you should pass the index in (here and to next_op()), rather than compute it by incrementing it on each opcode. I believe you have a bug where it isn't incremented when next_op() skips the update_offset() call.

Reply:
Indeed. New results:

Total: 230 errors; 10,136 files; 71,794 code objects; 3,364,493 lines; 4,841,077 opcodes; 4,769,283 opcode pairs; 2,263,338.0 cache_size; 770,108.0 cache wasted; 746,615 ops quickened; 135 skip quicken; 51,244 prev extended args